docs/server/source/quickstart.md

```diff
@@ -6,34 +6,55 @@ A. Install MongoDB as the database backend. (There are other options but you can
 
 [Install MongoDB Server 3.4+](https://docs.mongodb.com/manual/tutorial/install-mongodb-on-ubuntu/)
 
-B. Run MongoDB. Open a Terminal and run the command:
+Default database directory where MongoDB stores its data: /data/db.
+
+B. To run MongoDB with default database path i.e. /data/db, open a Terminal and run the following command:
+```text
+$ mkdir -p /data/db
```

Comment:
Without sudo, I get `mkdir: cannot create directory ‘/data’: Permission denied`. That is, change this line to:
```text
$ sudo mkdir -p /data/db
```